NBitcoin/IpExtensions.cs

```diff
@@ -183,7 +213,7 @@ public static byte[] GetGroup(this IPAddress address)
 				vchRet.Add((byte)(bytes[15 - 2] ^ 0xFF));
 				return vchRet.ToArray();
 			}
-			else if (address.IsTor())
+			else if (address.IsTor() || address.IsCjdns())

```

Comment:
IsI2P condition missing from this if-else, is that intentional?

Reply:
I2P addresses cannot be embedded in IP addresses so, it makes no sense to check for an impossible condition.

Reply:
ACK